README.md

```diff
@@ -1,2 +1,143 @@
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello World] <a name="about-project"></a>
+
+**[Hello World]** is a project to resolve the exercise named "Set up a 'Hello Microverse' project" of Microverse program, day 2.
+
+## 🛠 Built With HTML and CSS <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The project was developed using HTML and CSS.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Saying hello to the world.]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> www.helloworld.com
+
+- [Live Demo Link](https://www.helloworld.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> This project was elaborated as an exercise. It can be used as a guideline for professional Git use.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Oluwatosin Festus Ajibola
+> Martin Ossa
+> Tshepo Makama
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/festy1)
+
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/maov19)
+
+👤 **Author3**
+
+- GitHub: [@githubhandle](https://github.com/tshepo1)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[Goodbye World]**
+- [ ] **[Good Morning World]**
+- [ ] **[Good Night World]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Support our project by creating a Hello World project yourself.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank all the world.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Please this link is broken, kindly create a LICENSE file in your root folder, and customize [this text](https://gist.github.com/adamveld12/b64e74d14d7d9043b85c). Obviously, you've made a great effort but you are almost there. 👍

Reply:
LICENSE.md file was created, as requested.